docs/client/javascript-mono/_reactInstall.mdx

```diff
@@ -0,0 +1,34 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+
+## Installation
+
+To add support for React, first, add the Statsig JS client outlined [here](/client/javascript-sdk).
+
+Once the Statsig client is installed, you can add the React bindings as described below:
+
+<Tabs
+  defaultValue="npm"
+  groupId="js-install-choice"
+  values={[
+    {label: 'NPM', value: 'npm'},
+    {label: 'Yarn', value: 'yarn'},
+  ]}>
+
+  <TabItem value="npm">
+
+```shell
+npm install @statsig/react-bindings
```

Comment:
you need to npm install both.  We should document that explicitly rather than say "go here"